openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -525,5 +525,11 @@ jQuery(function () {
         import(/* webpackChunkName: "breadcrumb-select" */ './breadcrumb_select')
             .then(module => module.initBreadcrumbSelect(crumbs));
     }
+    
+    const loans = document.querySelectorAll('.leave');
```

Comment:
```suggestion
    const loans = document.querySelectorAll('#leave-waitinglist-dialog');
```

Lets use this id instead since `.leave` could be used elsewhere.